libGitWrap/Commit.cpp

```diff
@@ -347,23 +345,23 @@ namespace Git
 
     DiffList Commit::diffFromParent(Result& result, unsigned int index)
     {
-        GW_CD_CHECKED(Commit, DiffList(), result)
+        GW_CD_CHECKED(Commit, DiffList(), result);
```

Comment:
Actually just indentation complained about this :). As you know, this expands to something like `d = static_cast<...>(); if (...) {...}`.